Makefile

```diff
@@ -193,11 +193,12 @@ $(LOCALBIN):
 KUSTOMIZE ?= $(LOCALBIN)/kustomize
 CONTROLLER_GEN ?= $(LOCALBIN)/controller-gen
 ENVTEST ?= $(LOCALBIN)/setup-envtest
-GINKGO ?= $(LOCALBIN)/ginkgo
```

Comment:
isn't the GINKGO variable used below `ginkgo` target?

Reply:
my bad yea missed that let me fix that up

Reply:
done